daffodil-core/src/main/scala/org/apache/daffodil/dsom/SchemaComponent.scala

```diff
@@ -19,21 +19,10 @@ package org.apache.daffodil.dsom
 
 import scala.xml.Node
 import org.apache.daffodil.exceptions.Assert
-import org.apache.daffodil.xml.GetAttributesMixin
-import org.apache.daffodil.xml.NS
-import org.apache.daffodil.xml.XMLUtils
-import org.apache.daffodil.processors.NonTermRuntimeData
-import org.apache.daffodil.processors.RuntimeData
-import org.apache.daffodil.processors.VariableMap
-import org.apache.daffodil.processors.NonTermRuntimeData
-import org.apache.daffodil.xml.ResolvesQNames
-import org.apache.daffodil.schema.annotation.props.LookupLocation
-import org.apache.daffodil.schema.annotation.props.PropTypes
-import org.apache.daffodil.oolag.OOLAG._
+import org.apache.daffodil.processors.{RuntimeData, VariableMap, NonTermRuntimeData}
```

Comment:
Revert. This is just IntelliJ IDEA folding the imports into the scala { style } instead of line by line. 

It also may be  removing unneeded ones, which is ok, but we're not consciously converting everything to the other style.